### PR TITLE
Create README_zh.md

### PR DESCRIPTION
**Pull request by rocklau**

### CODE REVIEW

File: README_zh.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+- [与OpenAI的Azure Search - C#示例](https://github.com/Azure-Samples/azure-search-openai-demo-csharp/) - [文档](https://learn.microsoft.com/dotnet/azure/ai/get-started-app-chat-template?tabs=github-codespaces) | [公告帖子](https://devblogs.microsoft.com/dotnet/transform-business-smart-dotnet-apps-azure-chatgpt/) ChatGPT + 企业数据与Azure OpenAI和认知搜索（.NET）
+- [eShop参考应用程序](https://github.com/dotnet/eShop/)
+
+## 使用本地模型
+
+- [使用TorchSharp操作Phi2](https://github.com/LittleLittleCloud/Torchsharp-phi) - 一个演示如何在本地系统中访问Phi2模型的示例，使用.NET和[TorchSharp](https://github.com/dotnet/TorchSharp)库的帮助。
+- [使用TorchSharp操作Llama2](https://github.com/LittleLittleCloud/Torchsharp-llama) - 这个示例展示了如何使用.NET和[TorchSharp](https://github.com/dotnet/TorchSharp)库的帮助，在本地系统中使用Llama2模型。
+
+## 使用多模式模型
```
Comment (LittleLittleCloud) [inline README_zh.md:67]:
```suggestion
## 使用多模态模型
```

File: README_zh.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+- [用#30DaysOfAzureAI开始你的AI和.NET之旅](https://www.youtube.com/watch?v=567890)
+
+## 参考应用程序
+
+- [与OpenAI的Azure Search - C#示例](https://github.com/Azure-Samples/azure-search-openai-demo-csharp/) - [文档](https://learn.microsoft.com/dotnet/azure/ai/get-started-app-chat-template?tabs=github-codespaces) | [公告帖子](https://devblogs.microsoft.com/dotnet/transform-business-smart-dotnet-apps-azure-chatgpt/) ChatGPT + 企业数据与Azure OpenAI和认知搜索（.NET）
+- [eShop参考应用程序](https://github.com/dotnet/eShop/)
+
+## 使用本地模型
+
+- [使用TorchSharp操作Phi2](https://github.com/LittleLittleCloud/Torchsharp-phi) - 一个演示如何在本地系统中访问Phi2模型的示例，使用.NET和[TorchSharp](https://github.com/dotnet/TorchSharp)库的帮助。
```
Comment (LittleLittleCloud) [inline README_zh.md:64]:
```suggestion
- [Torchsharp-phi](https://github.com/LittleLittleCloud/Torchsharp-phi) - 一个基于Torchsharp实现的Phi模型。
```

File: README_zh.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+- [使用TorchSharp操作Llama2](https://github.com/LittleLittleCloud/Torchsharp-llama) - 这个示例展示了如何使用.NET和[TorchSharp](https://github.com/dotnet/TorchSharp)库的帮助，在本地系统中使用Llama2模型。
```
Comment (LittleLittleCloud) [inline README_zh.md:65]:
```suggestion
- [使用TorchSharp操作Llama2](https://github.com/LittleLittleCloud/Torchsharp-llama) -一个基于Torchsharp实现的llama-2模型。
```

File: README_zh.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+- [使用AI生成图像，使用Stable Diffusion、C#和ONNX运行时](https://devblogs.microsoft.com/dotnet/generate-ai-images-stable-diffusion-csharp-onnx-runtime/) - 关于如何使用ONNX运行时和.NET访问Stable Diffusion模型来生成图像的概述
+
+## 处理数据
+
+- [用.NET揭开检索增强生成功能的神秘面纱](https://devblogs.microsoft.com/dotnet/demystifying-retrieval-augmented-generation-with-dotnet/) - 关于如何在.NET中用称为检索增强生成的概念处理数据的详细指南。
+- [AugmentR](https://github.com/bradygaster/AugmentR) - 一个示例聊天机器人，展示了如何在.NET Aspire项目中使用Semantic Kernel，通过公共互联网URL的数据增强聊天。
+- [Vector Search AI助手](https://github.com/Azure/Vector-Search-AI-Assistant/tree/cognitive-search-vector) - 演示如何结合来自CosmosDB的数据，并通过认知搜索进行查询，创建AI搜索助手的应用程序。
+- [使用MongoDB的Vector Search AI助手](https://github.com/Azure/Vector-Search-AI-Assistant-MongoDBvCore) - 演示如何结合来自Azure CosmosDB for MongoDB的数据，并通过Azure OpenAI服务进行查询的应用程序。
+
+## 使用助手/代理
```
Comment (LittleLittleCloud) [inline README_zh.md:79]:
```suggestion
## 使用  Assistants/Agents
```

File: README_zh.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+# 教程和示例
+
+## 教程
+
+- [用AI构建你自己的课程助手](https://youtu.be/BRaltelZt6U?si=uuUvRc_9jSW4L601) "你在使用Azure OpenAI服务构建Copilot应用吗？你理解Copilot Stack的概念吗？在这次会议中，我们将使用Semantic Kernel构建您的第一个基于Copilot Stack的Copilot应用程序。" - 由Luis Quintanilla（微软）和Kinfey Lo（微软）主持
+- [用#30DaysOfAzureAI开始你的AI和.NET之旅](https://www.youtube.com/watch?v=567890)
+
+## 参考应用程序
+
+- [与OpenAI的Azure Search - C#示例](https://github.com/Azure-Samples/azure-search-openai-demo-csharp/) - [文档](https://learn.microsoft.com/dotnet/azure/ai/get-started-app-chat-template?tabs=github-codespaces) | [公告帖子](https://devblogs.microsoft.com/dotnet/transform-business-smart-dotnet-apps-azure-chatgpt/) ChatGPT + 企业数据与Azure OpenAI和认知搜索（.NET）
```
Comment (LittleLittleCloud) [inline README_zh.md:59]:
```suggestion
- [Azure Search with OpenAI - C#示例](https://github.com/Azure-Samples/azure-search-openai-demo-csharp/) - [文档](https://learn.microsoft.com/dotnet/azure/ai/get-started-app-chat-template?tabs=github-codespaces) | [公告帖子](https://devblogs.microsoft.com/dotnet/transform-business-smart-dotnet-apps-azure-chatgpt/) ChatGPT + 企业数据与Azure OpenAI和认知搜索（.NET）
```

File: README_zh.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+- [eShop参考应用程序](https://github.com/dotnet/eShop/)
```
Comment (LittleLittleCloud) [inline README_zh.md:60]:
```suggestion
- [eShop](https://github.com/dotnet/eShop/)
```

**Review by LittleLittleCloud — APPROVED** [review]
SFW with some minor suggestions for more accurate translation

**Comment by jmatthiesen** [comment]
Thanks for the submission @rocklau and thank you for the review @LittleLittleCloud!

@rocklau - please review the feedback and accept changes if they look OK to you. If I don't hear back by Friday I'll go ahead and accept all the edits and merge this in. Thanks again!

**Review by rocklau — COMMENTED** [review]
confirm.

**Review by LittleLittleCloud — APPROVED** [review]

**Pull request merged**